### PR TITLE
Lightgeists are gold core spawnable

### PR DESCRIPTION
**Pull request by ghost**
A lot of people love these things and this seems like a harmless change to give them what they want

### CODE REVIEW

**Comment by praisenarsie** [comment]
what are lightgeists

**Comment by Armhulen** [comment]
2 health mobs that can run around healing people, they can't speak any language but understand all languages

**Comment by JJRcop** [comment]
On local repo, ```git checkout master;git fetch https://github.com/tgstation/tgstation.git master;git reset --hard FETCH_HEAD;git push --force origin master```

**Comment by Armhulen** [comment]
very nice i'll do that when i can

File: code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm [hunk 1]
```diff
@@ -636,6 +636,7 @@ Difficulty: Very Hard
 	AIStatus = AI_OFF
 	stop_automated_movement = 1
 	var/heal_power = 5
+	gold_core_spawnable = 1
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm]: TRUE

Comment (Cyberboss) [reply code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm]: also new vars AFTER parent vars

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm (hunk 1 above)
Comment (Armhulen) [reply code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm]: hoo boy okay one second

**Comment by calzilla1** [comment]
Maybe ypu could slightly buff these ones

**Comment by jughu** [comment]
nah if you want strong revives you get magicarp :)

**Comment by davethwave** [comment]
I think they speak slime actually so if you have a PAI with the universal language module installed they could probably talk to them.

**Comment by DoctoreMad** [comment]
They speak their own language unless languages have been crunched lately

**Review by Cyberboss — APPROVED** [review]

**Comment by Cyberboss** [comment]
Changelog

**Comment by Repukan** [comment]
Does RnD really need healsluts though.

**Comment by Armhulen** [comment]
2 health rndsluts ok?

**Comment by XDTM** [comment]
you say that as if they couldn't just build a dispenser and dunk styptoc/silver sulf on their faces for the instaheal

**Comment by Repukan** [comment]
Sure, but making medicine for the different damages takes more than one click.

**Comment by Bawhoppen** [comment]
They have much more OP alternatives in like 30 different ways
(robotics medigun, R&D machines (sleepers, chems, clonerpods, cryo), INFINITE RESURRECTION/HEALING MAGICARP)

**Comment by davethwave** [comment]
And be aware that these things heal pretty slowly as well...

**Comment by Repukan** [comment]
If something is already broken that doesn't mean you should be free to hit it with a sledgehammer further.

**Comment by ExcessiveUseOfCobblestone** [comment]
kawaii jellyfish

**Comment by jughu** [comment]
Remember when i tried to remove magicarps from gold slime pool.

**Comment by Armhulen** [comment]
@jughu  soon

**Comment by Bawhoppen** [comment]
jughu has the right idea honestly

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-1). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Comment by AnonymousNow** [comment]
I know this is already in, but which pool is the lightgeist in? The blood pool, the plasma pool or the water pool?

**Comment by JJRcop** [comment]
@AnonymousNow There's only two pools: water, and plasma/blood. The lightgeists are in the plasma/blood pool.

**Comment by AnonymousNow** [comment]
Good to know, thanks.

I keep thinking of plasma/blood as one pool due to differing hostility, though I know that's not the case.

**Comment by AnonymousNow** [comment]
I've been in and out of xenobiology, spawning mobs from gold slime cores, ever since this was merged - and I haven't seen a single lightgeist. Are you sure this is working?

**Pull request merged**